GoogleForCreators/web-stories-wp
# Media hotlinking: use the hotlink endpoint

## Summary
Uses the new `hotlink` endpoint to get server-side validation for the inserted URL.


## Relevant Technical Choices



## To-do



## User-facing changes



## Testing Instructions
1. Open the Insert by Link dialogue.
2. Insert an Unsplash image link.
3. Verify the image is inserted correctly.
4. Insert Coverr video link
5. Verify the video is inserted correctly as a new element.
6. Try also with incorrect URLs (including .svg as that's not allowed) and verify it shows an appropriate message.




## Checklist




---



Fixes #8773
